mos-platform/commodore/cbm_k_ckout.s

```diff
@@ -0,0 +1,15 @@
+.include "imag.inc"
+.text
+
+;
+; unsigned char cbm_k_ckout(unsigned char FN);
+;
+.global cbm_k_ckout
+cbm_k_ckout:
+	tax
+	jsr __CKOUT
+	ldx #0

```

Comment:
The return type is unsigned char, so the value of X at return need not be defined. I think you can pull out the `ldx #0.`